vitalis-web/src/main/java/com/rocket/vitalis/model/Measurement.java

```diff
@@ -0,0 +1,54 @@
+package com.rocket.vitalis.model;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Delegate;
+
+import javax.persistence.*;
+import java.util.Date;
+
+/**
+ * Created by Ailin on 21/08/2016.
+ */
+@Entity
+public class Measurement {
+
+    @Id
+    @GeneratedValue
+    @Getter @Setter
+    private Long id;
+
+    @Column
+    @Getter @Setter
+    private Date measurementDate;
+
+    @Column(nullable = false)
+    @Getter @Setter @NonNull
+    private Double value;
+
+    @Column
+    @Getter @Setter
+    private MeasurementType measurementType;
+
+    /*DEBE ESTAR RELACIONADO CON Monitoring*/
+    @OneToOne
+    @Getter @Setter
+    private Monitoring idMonitoring;
+
+    public Measurement(String measureTypeString, Double value ){
+        switch (measureTypeString){
```

Comment:
esto se suele reemplazar con un método fromString en la clase MeasurementType, sería algo así

public static MeasurementType fromString(String code){
for (MeasurementType type: values()){
if(type.getCode().equals(code){
return type;
}
return null;
}

el return null puede también reemplazarse con un throw new IllegalArgumentException("Measurement type " + code +" unknown");